### PR TITLE
schema: fix `table not found` when `startTS` = 0

### PR DESCRIPTION
**Pull request by leoppro**
### What is changed and how it works?
when `startts=0`,  `LoadHistoryDDLJobs` function will return a empty jobs array, but we can't receive integrated DDL jobs from DDL Puller

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cdc/processor.go [hunk 1]
```diff
@@ -653,6 +653,7 @@ func (p *processor) addTable(ctx context.Context, tableID int64, startTs uint64)
 				if pEvent == nil {
 					continue
 				}
+				log.Info("sorter output", zap.Reflect("pevent", pEvent))
```
Comment (amyangfei) [inline cdc/processor.go:656]: Is this log necessary?

Comment (leoppro) [reply cdc/processor.go:656]: it's a debug code, I forgot to remove it.

**Pull request merged**